tests/sentry/api/endpoints/test_user_authenticator_enroll.py

```diff
@@ -35,9 +46,13 @@ def test_totp_can_enroll(self, validate_otp, email_log):
             kwargs={"user_id": "me", "interface_id": "totp"},
         )
 
-        resp = self.client.get(url)
+        with mock.patch("sentry.models.authenticator.generate_secret_key", return_value="Z" * 32):
+            resp = self.client.get(url)
+
         assert resp.status_code == 200
-        assert len(resp.data["qrcode"])
+        assert resp.data["secret"] == "Z" * 32
+        with io.open(get_fixture_path("totp_qrcode.json")) as f:
```

Comment:
Suggestion: instead of dealing with paths etc. why not make this function `provide_fixture` and do something like:
```py
with provide_fixture("totp_qrcode.json") as f:
```

Reply:
Yeah, that can be implemented along with https://github.com/getsentry/sentry/pull/17260#discussion_r383857230.